src/__tests__/cacao.test.ts

```diff
@@ -1,17 +1,32 @@
 import { Wallet } from '@ethersproject/wallet'
-import { Cacao, CacaoBlock } from '../cacao'
-import { SiweMessage } from '../siwe'
+import { Cacao, CacaoBlock } from '../cacao.js'
+import * as solanaWeb3 from '@solana/web3.js'
+import { SiweMessage } from '../siwx/siwe.js'
+import { SiwsMessage } from '../siwx/siws.js'
+import nacl from 'tweetnacl'
+import base58 from 'bs58'
```

Comment:
```suggestion
import { SiweMessage } from '../siwx/siwe.js'
import { SiwsMessage } from '../siwx/siws.js'
import { extractPublicKeyFromSecretKey, sign, verify } from '@stableliv/ed25519'
import u8a from 'uint8arrays'
```

Stabelib should handle all signing and uint8arrays can be used for encoding baseX strings.